src/database/sql_database.cc

```diff
@@ -2515,18 +2515,18 @@ void SQLDatabase::migrateMetadata(int objectId, const std::string& metadataStr)
         for (auto&& [key, val] : dict) {
             metadataSQLVals[quote(key)] = quote(val);
         }
-        auto fields = std::vector {
-            identifier("item_id"),
-            identifier("property_name"),
-            identifier("property_value"),
-        };
         for (auto&& [key, val] : metadataSQLVals) {
+            auto fields = std::vector {
+                identifier("item_id"),
+                identifier("property_name"),
+                identifier("property_value"),
+            };
             auto values = std::vector {
                 fmt::format("{}", objectId),
                 fmt::format("{}", key),
                 fmt::format("{}", val),
             };
-            insert(METADATA_TABLE, fields, values);
+            insert(METADATA_TABLE, std::move(fields), std::move(values));
```

Comment:
Move does not make sense to me if the target is a const ref. Recreating the vector in each loop seems expensive, too.